nextstrain_profiles/nextstrain-gisaid/builds.yaml

```diff
@@ -973,116 +973,87 @@ frequencies:
   reference:
     min_date: "2020-01-01"
     narrow_bandwidth: 0.038
-    recent_days_to_censor: 7
```

Comment:
> Additionally, this PR removes the build-specific setting for `recent_days_to_censor`. This setting wasn't actually being applied in a build-specific fashion and so we had been always using the workflow default of `0`. I decided to stick with this.

Good catch. These should also be removed from nextstrain_profiles/nextstrain-open/builds.yaml.

Relatedly, it would be good to update [Workflow config file reference](https://docs.nextstrain.org/projects/ncov/en/latest/reference/workflow-config-file.html) to distinguish between settings that are "global" (directly under `frequencies`) vs. build-specific. Currently one has to inspect this Snakemake logic to determine where it can be set:

https://github.com/nextstrain/ncov/blob/4477b9654a529c4840dbe5d006113bd8c33f73c2/workflow/snakemake_rules/common.smk#L203

Reply:
Thanks for the catch with other profiles. 05b4f0f3d6191be5a0fe4a888aaf5163f502db11 removes `recent_days_to_censor` for GISAID/21L and for open.

Reply:
With 64583576fb14120688210bfa62c45fe0c61b5a05, I've updated `workflow-config-file.rst` to describe what can be over-ridden via `default` and what can't.